tests/Pest.php

```diff
@@ -66,3 +66,8 @@ function paginationMockClient(string $prefix): MockClient
         }
     ]);
 }
+
+function superheroes(): array
```

Comment:
While it is a method to grab test data, I think it'd be nice to add array shape, as it could be used in tests for autocompletion.

```suggestion
/**
 * @returns array{
 *     id: int<1, max>,
 *     superhero: string,
 *     publisher: string,
 *     alter_ego: string,
 *     first_appearance: string,
 *     characters: string,
 * }
 */
function superheroes(): array
```